azavea/pwd-unitybar
# Enable host application to send custom nodes in custom app actions

## Overview

Enable host application to send custom nodes in custom app actions

Connects https://github.com/azavea/pwd-stormdrain-marking/issues/519

## Testing Instructions

Try out the staging site here and verify that everything still works as before:

https://deploy-preview-101--staging-pwd-unitybar.netlify.com/
